.gitignore

```diff
@@ -20,6 +20,7 @@
 .DS_Store
 *.pem
 .idea/
+.vscode/
```

Comment:
I don't use VS Code currently, but I was under the impression that the `.vscode/` folder could be used to share project specific settings (like formatOnSave etc.) and recommended extensions. By ignoring this folder we lose this, don't we?

Reply:
After configuring VS Code to use the .prettierrc file following this project, the local setting.json file, which is located in the .vscode folder, was initially added commited. To avoid pushing it, I added the folder to the .gitignore file. Since I did not find any .vscode direktory in the repo, I assumed this was ok fix for avoiding pushing unnecessary code.

Reply:
If it's the setting for which formatter to use for the language, I think it's okay to commit that actually. It enforces the correct formatter for other new developers (and current ones).

Reply:
Agreed! It adds value to have the .vscode directory. Can also add suggested extensions in that